packages/host/app/routes/card.gts

```diff
@@ -63,7 +41,7 @@ export default class RenderCard extends Route<Model | null> {
       : new URL('./', ownRealmURL);
 
     try {
-      let cardResource = getCard(this, () => url.href);
+      let cardResource = getCard(this, () => url.href); //TODO: this method creates a resource entangled with the route lifetime, but that is wrong since route instances live forever AND we are not using this in a resource-y way
```

Comment:
Is this a moment to try out [`ember-route-template`](https://github.com/discourse/ember-route-template)?

Reply:
Not in this PR, but definitely down to try it out.